edge/pkg/edged/edged_status.go

```diff
@@ -171,19 +155,10 @@ func (e *edged) getNodeStatusRequest(node *v1.Node) (*edgeapi.NodeStatusRequest,
 
 	for _, removedResource := range removedDevicePlugins {
 		// if the remmovedReousrce is not contained in the nameSet and contains specific tag
-		if !nameSet.Has(removedResource) {
-			status, err := retrieveDevicePluginStatus(removedResource)
-			if err == nil {
-				if node.Annotations == nil {
-					node.Annotations = make(map[string]string)
-				}
-				node.Annotations[apis.NvidiaGPUStatusAnnotationKey] = status
-				klog.Infof("Setting node annotation to add node status list to Scheduler")
-				continue
-			}
+		if nameSet.Has(removedResource) {
+			klog.Infof("Remove capacity for %s", removedResource)
+			delete(node.Status.Capacity, v1.ResourceName(removedResource))
```

Comment:
We should set the resource to 0 instead remove them? ref: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/nodestatus/setters.go#L349

Reply:
Ping  :)